packages/admin-ui-docs/docs/guidelines/components/button.mdx

```diff
@@ -1,131 +1,268 @@
 ---
 title: Button
 path: /button/
+hide_table_of_contents: true
 ---
 
-Buttons trigger an action, or allow the user to advance a state. They are where the user interacts with the screen, enabling actions or decisions.
-
-## Usage
+import { Page, Panel } from '@site/src/components/Page'
+import { PageSection } from '@site/src/components/PageSection'
+import { CodeSection } from '@site/src/components/CodeSection'
+import { MultipleCodeSection } from '@site/src/components/MultipleCodeSection'
 
-```jsx isStatic
-import { Button } from '@vtex/admin-ui'
-
-export function Example() {
-  return <Button onClick={() => {}}>Admin UI Button</Button>
+export const tabToc = {
+  overview: [
+    {
+      id: 'variants',
+      value: 'Variants',
+      level: 2,
+    },
+    {
+      id: 'how-does-each-variant-work',
```

Comment:
Why the `id`s and `level`s here?

Reply:
because they are necessary to create the link between the heading/anchor and the TOC (right navigation)